### DISCUSSION TITLE
Removing/replacing chevrons

### DISCUSSION THREAD

**Discussion by zambetti** [zellij-org/zellij#3022] [Q&A] [answered]
Maybe this is basic, but I've spotted this on the Zellij github...
https://github.com/zellij-org/zellij/issues/1402#issuecomment-1676381414
Does anyone know how this was achieved?
Particularly, the way the (misleading) directional chevrons were removed on the tabs and the bottom menu/tips.

![Screenshot 2023-12-19 at 15 33 28](https://github.com/zellij-org/zellij/assets/98515/78a21328-fa48-4de4-bc98-3e92ed3cb31d)

**Comment by cristiand391** [comment, marked as answer]
you can toggle those by setting the `simplified_ui` option:
https://zellij.dev/documentation/options#simplified_ui

**Reply by zambetti** [reply]
Thank you. This solved it, much improved.

I've been enjoying zellij so far (coming from tmux), but the choice of chevrons in this UI context is baffling since there is no directionality. I can only assume that they are an unfortunate aesthetic preference.

**Reply by cristiand391** [reply]
Yep, I also moved from tmux to zellij but never bothered to disable it and now it looks weird to me if I do 😄 

you can also use a custom status bar plugin, see:
https://github.com/dj95/zjstatus/

**Reply by zambetti** [reply]
Perhaps plugins can solve this, but what I don't understand about the UI design decisions of zellij, is that the default configuration is friendly to the point of being cumbersome; which is fine for the first few hours or days, but after quickly growing tired of the cruft, and choosing 'default_layout "compact"' then there's no helpful UI at all, even though there easily could be.

For example, in compact mode, I would expect that since the key-commands are chorded, when you press a significant modifier (eg: ctrl or alt), the possible options could then be somehow displayed either in the bar or in an overlay (unless locked). And if no modifiers are pressed, then there would be no (or minimal) visible UI. This would make compact mode as helpfully verbose as the default configuration, but only when requested/needed.

Where tmux was perhaps too opaque, with a steep learning curve, zellij seems to have gone either too far in the other direction (like nano), trying too hard to be GUI-like at the expense of what makes the terminal serene and pure. Or otherwise, in compact mode, just as opaque as tmux.

**Reply by imsnif** [reply]
Talk is cheap, friend. You have all the tools you need to write a plugin that behaves this way. Looking forward to seeing it :)

**Reply by zambetti** [reply]
💪🏼
